merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/GetSimulationResultsAction.java

```diff
@@ -97,9 +98,10 @@ public Map<String, List<Violation>> getViolations(final PlanId planId, final Sim
       throw new RuntimeException("Assumption falsified -- mission model for existing plan does not exist");
     }
 
-
     final var activities = new ArrayList<ActivityInstance>();
-    for (final var entry : results.simulatedActivities.entrySet()) {
+    final var simulatedActivities = this.simulationService.get(planId, revisionData,
+        r -> r.simulatedActivities).orElseGet(Map::of);
```

Comment:
I have a personal preference for `Collections::emptyMap` over `Map::of`, since the latter has overloads of multiple arity and you have to recognize that the nullary overload is being used here. (My first question when I see `Map::of` is "of what?", and it takes a half-second to realize there's nothing coming in from the caller.)

No big deal though.

Reply:
Good point, I considered using `.orElse(Map.of())` for that same clarity reason, but `Collections::emptyMap` is even better!

Reply:
`Map.of()` is cheap, so `orElse` would have been fine -- but `Collections::emptyMap` exists ^_^

Reply:
Switched to `Collections::emptyMap`.